src/main/java/view/planning/EventPlanningWindow.java

```diff
@@ -112,12 +112,19 @@ public String getEventName() {
 
     @Override
     public LocalDateTime getEventDate() {
-        return LocalDateTime.now(); //TODO replace with datepicker
+        Date tempDate = dateTimePicker.getDate();
```

Comment:
Don't name any variable `temp`, because local variables are temporary by definition. They are destroyed out of the scope.

Reply:
Ok so I renamed it as a "chosenDate"